be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -709,11 +709,19 @@ Status FragmentExecutor::prepare(ExecEnv* exec_env, const TExecPlanFragmentParam
         SCOPED_RAW_TIMER(&profiler.prepare_runtime_state_time);
         RETURN_IF_ERROR(_prepare_workgroup(request));
         RETURN_IF_ERROR(_prepare_runtime_state(exec_env, request));
+
+        auto query_mem_tracker = _query_ctx->mem_tracker();
+        SCOPED_THREAD_LOCAL_MEM_TRACKER_SETTER(query_mem_tracker.get());
```

Comment:
suggest to use FragmentContext.RuntimeState.instance_mem_tracker

Reply:
fixed.